src/components/implementation/no_interface/llbooter/boot_deps.h

```diff
@@ -47,7 +51,7 @@ boot_comp_pgtbl_expand(int n_pte, pgtblcap_t pt, vaddr_t vaddr, struct cobj_head
 	int i;
 	int tot = 0;
 	/* Expand Page table, could do this faster */
-	for (i = 0 ; i < (int)h->nsect ; i++) {
+	for (i = 0; i < (int)h->nsect; i++) {
```

Comment:
sigh.  Your argument hinges on me believing that you've read 90% of the code base.  Further, since parts of the code base that I know you know do put spaces on both sides (i.e. mbtests.c), it makes me doubt the 90% number even more.

The fact that a large number of the changes in this commit change ` ; ` to `; ` is evidence to the contrary.

Making arguments on feelings or a sense of something isn't going to be that convincing.  And you're a convincing guy, so you can do better :smiley: 

A way to make this argument that 1. the code base is inconsistent as it is (this is largely a function of how difficult it is to code review this as it is hard to see by scanning code), and 2. removing the space before is consistent with other parts of the style (I'll let you make that argument instead of giving you the examples).  The counter-argument is that the style guide explicitly says to use ` ; `, and much of the code base does use that style (a statement that has evidence in this PR).

That's a discussion to be had.  It is based on the subtlety of the situation, rather than on assertions.  That discussion instantly turns into a noop "no" if it is made using trivially discredited and unbelievable foundations.

Because you're generally awesome, I'll assume you're making the nuanced argument.  Can you provide some reasoning behind why `; ` would be more readable?  I'm absolutely willing to be convinced if we have an automatic way to update the code-base.